packages/terra-modal/src/Portal.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+
+/**
+ * The <Portal/> component render its children into a div and append the div to <body/>.
+ */
+const propTypes = {
+  /**
+   * Whether or not the children of <Portal/> should be rendered
+   */
+  isOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  isOpen: false,
+};
+
+class Portal extends React.Component {
```

Comment:
Can we investigate the use of something like `react-portal` directly? https://github.com/tajo/react-portal

I'd be happy to maintain less code if possible.